src/components/BankDetails/BankDetails.tsx

```diff
@@ -24,6 +24,7 @@ export default function BankDetails({ FormButtons, onSubmit }: Props) {
   const [currency, setCurrency] = useState<Currency>({
     code: "USD",
     name: "United States Dollar",
+    rate: 1,
```

Comment:
```suggestion
    rate: null,
```
nit: you don't need a rate in this component.

Reply:
value `1` won't hurt either to conform the `type Currency` or to say that rate of `$1` is obviously 1